### PR TITLE
Feature/ansible

### PR DESCRIPTION
**Pull request by erickeller**
refactor the initial playbook using parameter which can be overwritten using extra vars

### CODE REVIEW

File: ansible/TODO.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+* remove md5sum hash default database password
```
Comment (BRUNNEL6) [inline ansible/TODO.md]: This should be in an issue, shouldn't it?

**Comment by erickeller** [comment]
prefixed as discussed with semantic-release...

**Comment by BrunnerLivio** [comment]
is this ready to merge @erickeller?

**Comment by erickeller** [comment]
Could squash commits, rebased on master... Ready for merge.

**Pull request merged**